### DISCUSSION TITLE
Introduction

### DISCUSSION THREAD

**Discussion by Mouazzama-dev** [Vikash-8090-Yadav/SaL--dApp#89] [General]
Hey eveyone, I have done my Bachelors in Computer Science, graduated in 2022.  I am also  having 1.5 years working experience as a blockchain Developer.

Looking forward to contribute in this project. 
#HappyCoding

**Comment by Vikash-8090-Yadav** [comment]
Hey @Mouazzama-dev Thanks for choosing Sal-Dapp for the contribution. Hope u enjoyed the Ui and smart contract. Excited to see ur quality contribution in this repo. 
Good Luck :)
